app/src/componentsV2/BottomSheet/BottomSheet.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Tabs, Col, TabsProps } from "antd";
+import { RQButton } from "lib/design-system/components";
+import { useBottomSheetContext } from "./context";
+import { BiDockRight } from "@react-icons/all-files/bi/BiDockRight";
+import { BiDockBottom } from "@react-icons/all-files/bi/BiDockBottom";
+import { MdExpandLess } from "@react-icons/all-files/md/MdExpandLess";
+import { MdExpandMore } from "@react-icons/all-files/md/MdExpandMore";
+import "./BottomSheet.scss";
+
+export const BottomSheet: React.FC<TabsProps> = ({ items, defaultActiveKey }) => {
+  const { isBottomSheetOpen, isSheetPlacedAtBottom, toggleBottomSheet, toggleSheetPlacement } = useBottomSheetContext();
+
+  return (
+    <Col
+      span={isSheetPlacedAtBottom ? 24 : 11}
```

Comment:
This still depends on sheet placement. What if the bottomsheet is not placed within a Row component? Wouldn't this break